src/turbomind/layers/sampling_layers/TopPSamplingLayer.cu

```diff
@@ -342,16 +273,6 @@ void TopPSamplingLayer<T>::runSampling(TensorMap* output_tensors, TensorMap* inp
         skip_decode_buf_ + ite * local_batch_size);
     sync_check_cuda_error();
 
-    invokeComputeToppDecay(
```

Comment:
Since the call to this kernel is no longer needed, we can consider deleting the corresponding implementation as well.
https://github.com/InternLM/lmdeploy/blob/5059f37c1db6f607c0723932f8f90834ae6b7ba2/src/turbomind/kernels/sampling_topp_kernels.h#L153
https://github.com/InternLM/lmdeploy/blob/5059f37c1db6f607c0723932f8f90834ae6b7ba2/src/turbomind/kernels/sampling_topp_kernels.cu#L1581